backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/TorClient.kt

```diff
@@ -57,11 +59,22 @@ class TorClient private constructor(
      *
      * Each connection returned by this method is isolated from any other Tor usage.
      */
-    suspend fun connectToLightwalletd(endpoint: String): TorLwdConn =
+    suspend fun createWalletClient(endpoint: String): PartialTorWalletClient =
+        accessMutex.withLock {
+            withContext(Dispatchers.IO) {
+                checkNotNull(nativeHandle) { "TorClient is disposed" }
+                IsolatedTorWalletClient.new(
+                    isolatedTorClient = isolatedClientInternal(),
+                    endpoint = endpoint
+                )
+            }
+        }
+
+    suspend fun connectToLightwalletd(endpoint: String): Long =
```

Comment:
We absolutely do not want a native handle being exposed from a public `TorClient` method, because then there is nothing ensuring that it is properly handled per its safety requirements. It must be wrapped like it currently is:
```suggestion
    suspend fun connectToLightwalletd(endpoint: String): TorWalletClient =
```